Assets/SEE/Utils/Raycasting.cs

```diff
@@ -106,7 +106,14 @@ public static bool RaycastAnything(out RaycastHit raycastHit, float maxDistance
         {
             raycastHit = new RaycastHit();
             Physics.queriesHitBackfaces = true;
-            return !IsMouseOverGUI() && Physics.Raycast(UserPointsTo(), out raycastHit, maxDistance);
+            if (SceneSettings.InputType == PlayerInputType.DesktopPlayer)
+            {
+                return !IsMouseOverGUI() && Physics.Raycast(UserPointsTo(), out raycastHit, maxDistance);
+            }
+            else
+            {
+                return Physics.Raycast(UserPointsTo(), out raycastHit, maxDistance);
+            }
```

Comment:
In VR the method "IsMouseOverGUI()" should not be accessed, because it is currently not used there and only displays an error in the console, therefor I would like to keep it this way.

Reply:
Makes sense, but the simplified version compiles to the same code as the original version. The boolean `||` operator is short-circuiting, meaning that `IsMouseOverGUI()` will never be called if the player isn't the desktop player. Still, if you prefer the original version for readability reasons, you can keep it that way if you want.